.eslintrc.json

```diff
@@ -21,6 +21,14 @@
 		"jsdoc"
 	],
 	"rules": {
+    "@typescript-eslint/consistent-type-imports": [
+       "error",
+       {
+        "disallowTypeAnnotations": true,
+        "fixStyle": "separate-type-imports",
```

Comment:
There's an alternative to this which adds `type` to each individual import... this adds `type` to the entire import line. I don't care which we use.